orbit/pkg/table/fleetd_logs/fleetd_logs.go

```diff
@@ -134,6 +135,11 @@ func processLogEntry(event []byte) ([]Event, error) {
 		var sqliteTime string
 		evtTime, ok := evt["time"].(string)
 		if ok {
+			// UTC times sometimes end in Z instead of time offset.
+			// Replace it with a regular offset so we can parse it
+			if strings.HasSuffix(evtTime, "Z") {
+				evtTime = evtTime[:len(evtTime)-1] + "+00:00"
+			}
```

Comment:
any reason to not use a better fitting template instead? eg:

```
goTime, err := time.Parse(time.RFC3339, evtTime)
```

here's a [playground](https://go.dev/play/p/D4pLmYHywcz) where you can test it

Reply:
I don't think you're missing anything, I didn't notice that they had defined a constant for this format. 🙂
I'll swap it out and see if it continues to work